guides/micronaut-cloud-database-azure/micronaut-cloud-database-azure.adoc

```diff
@@ -0,0 +1,197 @@
+external:micronaut-cloud-database-base/start.adoc[]
+
+We will then connect the application to https://azure.microsoft.com/en-us/services/mysql/[Azure Database for MySQL].
```

Comment:
this should probably go after the account note so it doesn't split up the reqs list, or we could rework start.adoc